tools/accuracy_checker/accuracy_checker/postprocessor/super_resolution_image_recovery.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright (c) 2018-2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cv2
+import numpy as np
+from PIL import Image
+
+from ..config import StringField, NumberField
+from .postprocessor import Postprocessor
+from ..representation import SuperResolutionPrediction, SuperResolutionAnnotation
+
+
+class SRImageRecovery(Postprocessor):
+    __provider__ = 'sr_image_recovery'
+
+    annotation_types = (SuperResolutionAnnotation, )
+    prediction_types = (SuperResolutionPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'target_color': StringField(optional=True, choices=['bgr', 'rgb'], default='rgb'),
+            'input_size': NumberField(
```

Comment:
what if they are not equal?

Reply:
BTW, you already have info about input shape in image_meta, so you can extract it from it withoud additional parameters.

Reply:
fixed. we need to get not size of image, but size of input of model